### ISSUE TITLE
Keyboard nav and focus

### ISSUE THREAD

**Issue by swegele** [stavroskasidis/BlazorContextMenu#66]
When a context menu opens, it is really helpful to keyboard nav ready to work which usually requires focus to be on the element.  Currently the first menu item does not have focus and esc key does not work.

Would be nice if, when the menu opens, the following is true:
- focus is on first clickable menu item
- up/down keys work to navigate menu if it has focus and if parent/child item focused then left/right keys work too to focus relevant item
- escape key closes menu and returns focus to element that was active at the time of menu opening

I've looked at other blazor components and no one seems to have this feature yet...makes me wonder if there is some reason it is not do-able?

**Comment by stavroskasidis** [comment]
This is something that I definitely want to implement. 
Even though it seems that it requires a bit of work, I don't think that it is not do-able.

No eta though, sorry.
